src/common/state/QueryList.C

```diff
@@ -1195,7 +1195,7 @@ QueryList::QueryExists(const std::string &name, QueryType t)
     bool match = false;
     for (size_t i = 0; i < names.size(); i++)
     {
-        if (StringHelpers::CaseInsenstiveEqual(name,names[i]))
+        if (StringHelpers::CaseInsensitiveEqual(name,names[i]))
```

Comment:
Sorry I didn't catch this earlier. There is a .code file that should be modified.

Reply:
Oh, crap :hankey:. Forgot to regen that now too after you asked for the other. Fixed now. That was only change.